### PR TITLE
boa: move boa to imgcook/boa

### PR DESCRIPTION
**Pull request by FeelyChau**
`BOA` has beed moved into the org `imgcook`: [https://github.com/imgcook/boa](https://github.com/imgcook/boa).

### CODE REVIEW

**Comment by yorkie** [comment]
The CodeCoverage seems to be lower :(

**Comment by FeelyChau** [comment]
> The CodeCoverage seems to be lower :(

We need to fill the tests ASAP.

File: example/pipelines/rnn.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "specVersion": "2.0",
+  "dataSource": "file:///Users/feiyu.zfy/work/my-datasource/build/script.js?digits=2&numExamples=100",
```
Comment (yorkie) [inline example/pipelines/rnn.json:3]: Is this related?

Comment (FeelyChau) [reply example/pipelines/rnn.json]: Removed.

**Comment by yorkie** [comment]
And boa examples should be moved, too.

**Comment by yorkie** [comment]
And imgcook/boa's build is also required before landing this.

**Comment by FeelyChau** [comment]
> And boa examples should be moved, too.

See: https://github.com/imgcook/boa/pull/3

**Comment by FeelyChau** [comment]
> And imgcook/boa's build is also required before landing this.

See: https://github.com/imgcook/boa/pull/4

**Comment by FeelyChau** [comment]
Boa 2.1.0 released, we could merge this PR now. @yorkie

**Review by yorkie — APPROVED** [review]
